browser-test/src/applicant/navigation/northstar_application_navigation_common_intake.test.ts

```diff
@@ -223,8 +209,6 @@ test.describe('Applicant navigation flow', {tag: ['@northstar']}, () => {
       await validateScreenshot(
```

Comment:
**suggestion:** Here too, we can consider just validating that the submitted program shows vs taking the screenshot.